docs/deploy/aws.md

```diff
@@ -49,7 +49,7 @@ If you already have a database, take note of the connection string and skip to t
    blockstore:
      type: s3
      s3:
-       region: us-east-1
+       region: us-east-1 # optional, fallback in case discover from bucket is not supported
```

Comment:
This can be confusing for AWS S3 users.  Maybe
```suggestion
       # Uncomment on non-AWS S3 compatible implementations which do not support GetBucketLocation (e.g. MinIO).
       # region: us-east-1
```

Reply:
In minio we will also need to add `discover_bucket_region: false`, maybe it will be better off without it.

Reply:
Please feedback here and I'll update the docs in a different PR.